src/CalendarDate.ts

```diff
@@ -34,6 +34,14 @@ export class CalendarDate {
    */
   readonly weekday!: number;
 
+  /**
+   * Customizes the default string description for instances of `CalendarDate`.
+   * @returns {string} The string 'CalendarDate' which indicates the custom type of the object.
+   */
+  get [Symbol.toStringTag]() {
+    return 'CalendarDate';
+  }
```

Comment:
```suggestion
  /**
   * Customizes the default string description for instances of this class.
   */
  get [Symbol.toStringTag](): string {
    return 'CalendarDate';
  }
```
all the other methos dont define @returns, so i would like to keep it that way.